lamin_cli/compute/modal/_modal_image.py

```diff
@@ -0,0 +1,102 @@
+import modal
+
+"""
+Maybe some abstraction of creating images needs to be done, its doable we just need agreement, I am not using this now but its mock code. 
+"""
+
+
+def create_image(
+    base_type="debian_slim",
+    python_version="3.10",
+    pip_packages=None,
+    apt_packages=None,
+    env_vars=None,
+    commands=None,
+    local_dirs=None,
+    local_files=None,
+    local_python_modules=None,
+    force_rebuild=False,
+    gpu_type=None
+):
+    """
+    Create a customized Modal image based on specified parameters.
+    
+    Parameters:
+    -----------
+    base_type : str
+        The base image type to use. Options: "debian_slim", "micromamba", or a registry URL.
+    python_version : str
+        Python version to use (e.g., "3.10").
+    pip_packages : list
+        List of Python packages to install with pip.
+    apt_packages : list
+        List of system packages to install with apt.
+    env_vars : dict
+        Environment variables to set in the image.
+    commands : list
+        Shell commands to run during image building.
+    local_dirs : dict
+        Dictionary mapping local directory paths to remote paths.
+    local_files : dict
+        Dictionary mapping local file paths to remote paths.
+    local_python_modules : list
+        List of local Python modules to add to the image.
+    force_rebuild : bool
+        Whether to force rebuilding the image.
+    gpu_type : str
+        GPU type to use during image setup (e.g., "H100").
+        
+    Returns:
+    --------
+    modal.Image
```

Comment:
You can drop this and just annotate with a return type instead. It will document it.